app/views/performance_dashboards/overview/_overall.haml

```diff
@@ -1,38 +1,49 @@
-%h2 Overall
-.well
+%h2.c-heading.c-heading--full-bleed Overall Summary
+.o-container--full-bleed.o-container--featured
+
   .table-responsive
     %table.table
       %thead
         %tr
-          %th
-          %th
-            Homeless
+          %th.border-top-0
+          %th.border-top-0
+            %span.font-weight-bold Homeless
             %i.icon-info{data: {toggle: :tooltip, title: 'Clients meeting the above criteria who were enrolled in ES, SH, SO, or TH during the period.'}}
-          %th
-            Literally Homeless
+          %th.border-top-0
+            %span.font-weight-bold Literally Homeless
             %i.icon-info{data: {toggle: :tooltip, title: 'Clients meeting the above criteria who were enrolled in ES, SH, or SO during the period.'}}
-          %th
-            Newly Homeless
+          %th.border-top-0
```

Comment:
Should we standardize these as `bt-0`?  Similar to `pt-0`?

Reply:
I suppose we could. `border-top-0` is bootstrap's pattern for border utilities: https://getbootstrap.com/docs/4.0/utilities/borders/
I'm not sure why they didn't use the `b{direction}-{amount}` convention.

Reply:
Ah, let's leave it then.  Thanks